digdag-tests/src/test/java/acceptance/PgIT.java

```diff
@@ -199,6 +199,73 @@ public void selectAndDownloadWithNullValues()
     }
 
     @Test
+    public void selectAndStoreResult()
+            throws Exception
+    {
+        copyResource("acceptance/pg/select_store_result.dig", root().resolve("pg.dig"));
+        copyResource("acceptance/pg/select_table.sql", root().resolve("select_table.sql"));
+
+        setupSourceTable();
+
+        CommandStatus status = TestUtils.main(
+                "run", "-o", root().toString(),
+                "--project", root().toString(),
+                "-p", "pg_database=" + tempDatabase,
+                "-p", "outfile=out",
+                "pg.dig");
+        assertThat(status.code(), is(0));
```

Comment:
Adding first message argument to assertThat is very helpful for debugging:

```
assertThat(status.errUtf8(), status.code(), is(0));
```

This shows `status.errrUtf8()` when the assertion fails. It informs reason of the failure which is helpful especially on CI.